docs/creating-studies.md

```diff
@@ -127,16 +147,27 @@ styling.
   to find other problems if you lightly adhere to this from the start.
   - Agggregate count tables should have the first word after the study prefix be
   `count`, and otherwise the word `count` should not be used.
+
+**Metadata tables**
   - Creating a table called `my_study__meta_date` with two columns, `min date` and
   `max date`, and populating it with the start and end date of your study, will
   allow other Cumulus tools to detect study date ranges, and otherwise bakes the
   study date range into your SQL for future reference.
+  - Creating a `my_study__meta_version` with one column, `data_package_version`, and
+  giving it an integer value as shown in this snippet:
+  ```sql
+  CREATE TABLE my_study__meta_version AS
+  SELECT 1 AS data_package_version;
+  ```
+  allows you to signal versions for use in segregating data upstream, like in the
+  Cumulus aggregator - just increment it when you want third parties to start running
+  a new data model. If this is not set, the version will implicitly be set to zero.
 
 ## Sharing studies
 
-If you want to share your study as part of a publication, you'll need to open a PR - 
-after cloning this repository, make a branch, and add your study config to the
-`cumulus_library/studies/` directory, and then just open a PR. 
+If you want to share your study as an official Cumulus study, please let us know
+via the [discussion forum](https://github.com/smart-on-fhir/cumulus/discussions) -
+we can talk more about what makes sense for your use case.
```

Comment:
Nice :+1:  - safer to open a dialog vs a PR that might create an expectation that their code will make it in.

Reply:
gotta drive traffic to those forums too.